listenbrainz/webserver/templates/user/music_services.html

```diff
@@ -72,32 +72,9 @@ <h3 class="panel-title">Youtube</h3>
             <div class="panel-body">
                 <p>
```

Comment:
I wonder if we should allow people to disable playing from YT as a preference.

Reply:
@MonkeyDo had some plans about implementing preferences in BrainzPlayer. Maybe revisit when we do that?

Reply:
Yeah, perfect.